README.md

```diff
@@ -65,6 +65,9 @@ can browse locally to see API docs on the various packages.
 
 ### Libusb fails to build, why?
 
+If you are running on linux, you may not have the proper dependencies installed. For ubuntu, try the
+following: `sudo apt-get install libudev1 libudev-dev`. These are needed to compile the usb driver.
```

Comment:
travis has more ```sudo apt-get install -y libudev-dev libusb-1.0-0 libusb-1.0-0-dev```
i think maybe the other stuff can help even more?

Reply:
Good point, thank you for the careful eye.